tests/Fixer/Semicolon/SemicolonAfterInstructionFixerTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Semicolon;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ */
+final class SemicolonAfterInstructionFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideCases
+     * @requires PHP 5.4
```

Comment:
- no need for require 5.4 when only one case is failing on 5.3 and all the rest are working
- so, we test whole fixer only on 5.4+, but run it on 5.3+. then, on 5.3 we have no idea how the fixer will behave (as it still runs, but we dont check how)
- isn't whole thing more about `short_open_tag` instead of PHP version ?